slack_done.gs

```diff
@@ -102,26 +107,21 @@ function done_process_modal(args){
   var access_token = PropertiesService.getScriptProperties().getProperty('ACCESS_TOKEN'); // confidential Slack API access token
 
   // process done command
-  var out_message = done(args);
+  // Note: done returns JSON.stringify({blocks:[...]})
+  // We need to unpack and repack to mark it as ephemeral.
+  var payload = JSON.parse(done(args));
+  payload["response_type"] = "ephemeral";
+  payload = JSON.stringify(payload);
 
   // send reply to slack user with response_url functionality
   var response_url = args.response_url;
-  var options = {
-    method: "post",
-    contentType: 'application/json; charset=utf-8',
-    headers: {Authorization: 'Bearer ' + access_token},
-    payload: JSON.stringify({
-      "text": out_message,
-      "response_type": "ephemeral"
-    })
-  };
-  var return_message = UrlFetchApp.fetch(response_url, options).getContentText(); // Send post request to Slack response_url
+  var return_message = postToSlack(payload, response_url)
```

Comment:
is the response_url reply still required here? or is it now handled in handle_async function?

Reply:
Good point.

Because of the reply url issue you mentioned for slack, I haven't been able to test the "done modal". 

But I suggest the following changes below:
- Add a switch in handleSlackCommand for just done_process_modal so that it return an empty string.
- Merge `done_process_modal` with `done`

What do you think?